messaging/broker.go

```diff
@@ -57,6 +57,14 @@ func (b *Broker) metaPath() string {
 	return filepath.Join(b.path, "meta")
 }
 
+// Index returns the highest index seen by the broker.
```

Comment:
I guess somewhere we need to capture that the significance of this is that highest across all topics has been safely applied in the Raft log, if I remember correctly.